cesiumTestsExtension/bindings/PythonBindings.cpp

```diff
@@ -0,0 +1,20 @@
+#include <carb/BindingsPythonUtils.h>
+#include "CesiumTests.h"
```

Comment:
Looks like this file isn't formatted correctly. Could you run the formatter? (In VSCode, Ctrl + Shift + B then Format)

Note you will need to update `LINT_SOURCE_DIRECTORIES` for this work.